src/wazuh_modules/vulnerability_scanner/qa/test_false_positive_data/001/expected_002.out

```diff
@@ -0,0 +1,3 @@
+[
+    "Match found, the package 'grafana', is vulnerable to 'CVE-2023-4822'. Current version: '8.5.6' (less than '9.4.16' or equal to ''). - Agent '' (ID: '001', Version: '')."
```

Comment:
If the content version changes, this test will pass any ways. Lets make it less precise
```suggestion
    "Match found, the package 'grafana', is vulnerable to 'CVE-2023-4822'. Current version: '8.5.6'"
```